go.mod

```diff
@@ -14,6 +14,7 @@ replace (
 
 	github.com/rancher/rancher/pkg/apis => ./pkg/apis
 	github.com/rancher/rancher/pkg/client => ./pkg/client
+	github.com/rancher/shepherd => github.com/anupama2501/shepherd v0.0.0-20240304185105-f245828c18fe
```

Comment:
just a note; we should keep a PR in `draft` state until the rancher/shepherd PR has merged. Otherwise, its risky to accidentally merge a PR in rancher that is pointing to someone else's branch

Reply:
Updated the header.